data/coalition/coalition missions.txt

```diff
@@ -146,6 +146,77 @@ mission "Coalition License Patch"
 
 
 
+mission "lost in coalition"
+	landing
+	invisible
+	source
+		government "Coalition" "Heliarch"
+	to offer
+		has "Coalition: Contributor: done"
+	on offer
+		require "Jump Drive" 0
+		event "lost in coalition" 14
+		fail
+
+event "lost in coalition"
+
+
+
+mission "Coalition: Lost: Marker"
+	name "Foreign Aid"
+	description "The Heliarchs have noticed that you've lost your jump drive, being now stranded in Coalition space, and have offered to help. Head to <destination> to speak with them."
+	landing
+	source
+		near "Ekuarik" 1 100
+	destination "Ring of Wisdom"
+	to offer
+		has "event: lost in coalition"
+	on offer
+		conversation
+			`Another landing, another day spent stranded in this alien region, with no other humans in sight. Be it with the help of a Heliarch agent, via their own translation devices, or just by gesturing and hoping for the best, you've managed to get by just fine, but the thought that you may live the rest of your life living among an alien civilization refuses to leave your mind.`
+			`	Trying to take your mind off of it all, you start walking around <origin>, looking for some place to eat, but are stopped by a Heliarch delegation shortly after you leave your ship.`
+			`	"Cautiously observed during the past few weeks, scans of your ship have been, Captain <last>," the Saryd begins.`
+			`	"Unlike previous scans, a jump drive in your ship, our scanners found not," the Kimek continues.`
+			`	"Though appreciate the work you here perform, we do, for you to be cut-off from the galaxy, unlike the Quarg we wish not," the Arach says. "Discussed much, our elders have, and decided to help you, they have. To <destination> you must go, there discussed, the terms will be."`
+			`	You thank them for bringing you this information, and go back to your ship to prepare a route to <destination>.`
+	to complete
+		has "Coalition: Lost: Offer: active"
```

Comment:
I'd instead change this to never complete, then have accepting the next mission fail this one.
```suggestion
		never
```